Cargo.toml

```diff
@@ -296,6 +296,7 @@ similar-asserts = "1.4.2"
 slotmap = { version = "1.0.6", features = ["serde"] }
 smallvec = { version = "1.0", features = ["const_generics", "union"] }
 static_assertions = "1.1"
+stl_io = { git = "https://github.com/oxkitsune/stl_io", branch = "gijs/mesh-name" }
```

Comment:
given activity on that crate, do you think this is going to land and be released soon? Otherwise we have to get it out ourselves as it's blocking our own release